unity-client/Assets/Builder/Scripts/DCLBuilderObjectSelector.cs

```diff
@@ -152,14 +145,14 @@ private void OnMouseUp(int buttonId, Vector3 mousePosition)
             {
                 return;
             }
-            if (entityEnqueueForDeselectInfo.pressedEntity != null)
+            if (lastPressedEntityInfo.pressedEntity != null)
             {
-                if ((Time.unscaledTime - entityEnqueueForDeselectInfo.pressedTime) < MAX_SECS_FOR_CLICK)
+                if ((Time.unscaledTime - lastPressedEntityInfo.pressedTime) < MAX_SECS_FOR_CLICK)
```

Comment:
Please add a comment for this as well, as its not very clear of the intention of having a time threshold.